addons/web_tip/static/src/js/tip.js

```diff
@@ -250,6 +250,10 @@ var Tip = Class.extend({
                     def.resolve();
                 }
             });
+            self.$element.on('mousedown', function($ev) {
+                self.end_tip(tip);
+                def.resolve();
+            });
```

Comment:
@jpa-odoo Don't do anything, I did it for you. But recall that it is better to try to fix the data BEFORE thinking to change the generic framework